equinix/data_source_network_device.go

```diff
@@ -661,6 +661,19 @@ func createDataSourceVendorConfigurationSchema() map[string]*schema.Schema {
 			Sensitive:   true,
 			Description: neDeviceVendorConfigDescriptions["LicenseID"],
 		},
+		neDeviceVendorConfigSchemaNames["PanoramaIPAddress"]: {
+			Type:        schema.TypeString,
+			ForceNew:    true,
```

Comment:
We don't need forceNew tag for computed attributes.

Reply:
addressed